queries/models/projeto_subsidio_sppo_encontro_contas/aux_balanco_rdo_servico_dia.sql

```diff
@@ -73,27 +57,23 @@ sumario_dia AS (
     SUM(km_apurada) AS km_subsidiada,
     sum(valor_subsidio_pago) as subsidio_pago
   FROM
-    {{ ref("sumario_servico_dia_historico") }}
-    -- `rj-smtr.dashboard_subsidio_sppo.sumario_servico_dia_historico`
+    {# {{ ref("sumario_servico_dia_historico") }} #}
+    `rj-smtr.monitoramento.sumario_servico_dia_historico`
   WHERE
-    DATA BETWEEN "2022-06-01"
-    AND "2023-12-31"
-    and valor_subsidio_pago = 0
+    DATA BETWEEN "{{ var('start_date') }}"
+    AND "{{ var('end_date') }}"
+    {# and valor_subsidio_pago = 0 -- Desabilitar para Cenário E #}
   GROUP BY
     1,
     2,
     3),
 rdo_filtrada as (
-    select rdo.* from
-    (
-      select * from rdo
-      left join servico_dia_atipico sda
-      using (data, servico)
-      where sda.data is null
-    ) rdo
-    left join sumario_dia sd
+    select rdo.* from rdo
+    {# left join sumario_dia sd #}
+    full join sumario_dia sd -- Cenário E1
     using (data, servico)
-    where sd.servico is null
+    {# where sd.servico is null #}
+    where (sd.servico is distinct from rdo.servico) or rdo.receita_tarifaria_aferida = 0 -- Cenário E1

```

Comment:
Não muda nada, haja vista o filtro aplicado anteriormente

```suggestion
    where sd.servico is distinct from rdo.servico -- Cenário E1

```